steps/wetransfer-upload/v0.0.2/step.yml

```diff
@@ -0,0 +1,52 @@
+title: wetransfer-upload
```

Comment:
step's title will appear on the workflow editor UI, please use: `WeTransfer` as title instead

Reply:
Ok